apps/app/src/app/globals.css

```
@@ -1,5 +1,7 @@
 @import "@repo/ui/globals.css";
 
+@plugin '@tailwindcss/typography';
```

Stylelint (17.4.0)

[error] Unexpected unknown at-rule "@plugin" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)